MREUnityRuntime/MREUnityRuntimeLib/Core/ActorManager.cs

```diff
@@ -25,15 +25,44 @@ internal class ActorManager : ICommandHandlerContext
 
 		internal Dictionary<Guid, Actor>.ValueCollection Actors => _actorMapping.Values;
 
+		public delegate void RigidBodyAddedHandler(Guid id, UnityEngine.Rigidbody rigidbody, bool isOwned);
+		internal event RigidBodyAddedHandler RigidBodyAdded;
+
+		public delegate void RigidBodyRemovedHandler(Guid id);
+		internal event RigidBodyRemovedHandler RigidBodyRemoved;
+
+		public delegate void RigidBodyGrabbedHandler(Guid id, bool isGrabbed);
+		internal event RigidBodyGrabbedHandler RigidBodyGrabbed;
+
 		internal ActorManager(MixedRealityExtensionApp app)
 		{
 			_app = app;
 		}
 
+		private void OnRigidBodyAdded(Guid id, UnityEngine.Rigidbody rigidbody, bool isOwned)
+		{
+			RigidBodyAdded?.Invoke(id, rigidbody, isOwned);
+		}
+
+		private void OnRigidBodyRemoved(Guid id)
+		{
+			RigidBodyRemoved?.Invoke(id);
+		}
+
+		private void Actor_RigidBodyGrabbed(Guid id, bool isGrabbed)
+		{
+			RigidBodyGrabbed?.Invoke(id, isGrabbed);
+		}
+
 		internal Actor AddActor(Guid id, Actor actor)
 		{
 			actor.Initialize(id, _app);
 			_actorMapping[id] = actor;
+
+			actor.RigidBodyAdded += OnRigidBodyAdded;
+			actor.RigidBodyRemoved += OnRigidBodyRemoved;
+			actor.RigidBodyGrabbed += Actor_RigidBodyGrabbed;
```

Comment:
Naming consistency: Rename `Actor_RigidBodyGrabbed` -> `OnRigidBodyGrabbed` (or the other way around).

Reply:
good catch, thanks